client/src/components/EquipLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from "@chakra-ui/react";
+
+interface Props {
+  equipItemLink?: string;
```

Comment:
just make the prop required - doesn't make any sense to render this w/o a link. you also should be able to make it just take an item.